submissions/f3d0t/task_js_dom/script.js

```diff
@@ -44,34 +44,39 @@ const createMenuLinks = (array, container) => {
 
 const getPokemonContent = (id) => {
 	const wrapper = document.createElement("div");
-	const item = pokemons.find((item) => item.id == id);
-	const h2 = document.createElement("h2");
-	h2.classList.add("main__name");
-	h2.innerText = item.name;
-	const img = document.createElement("img");
-	img.classList.add("main__img");
-	img.src = item.imagePath;
-	img.alt = item.name;
-	const p = document.createElement("p");
-	p.classList.add("main__text");
-	p.innerText = item.description;
-	wrapper.append(h2, img, p);
-	return wrapper;
+	const currentPokemon = pokemons.find((item) => item.id == id);
+	if (currentPokemon !== undefined) {
```

Comment:
```suggestion
	if (!currentPokemon) {
```
Also:
> My suggestion: pass only item to this function. (If there is not such item don't call this function)

[Old comment](https://github.com/kottans/frontend-2021-homeworks/pull/390#discussion_r550232175)

Why this is good: you prevent redundant function calls.